src/backend/utils/mmgr/aset.c

```diff
@@ -262,7 +262,7 @@ static void AllocSetFreeHeader(MemoryContext context, void *pointer);
 static void *AllocSetRealloc(MemoryContext context, void *pointer, Size size);
 static void AllocSetInit(MemoryContext context);
 static void AllocSetReset(MemoryContext context);
-static void AllocSetDelete(MemoryContext context);
+static Size AllocSetDelete(MemoryContext context);
```

Comment:
I think it is not good idea to change postgres api

Reply:
I do it like it is at greenplum master.